src/main/resources/com/ibm/vCard/vCard_Format_2.1.xsd

```diff
@@ -108,19 +110,20 @@
        						 textZonedSignStyle="asciiStandard" 
        						 trailingSkip="0" 
        						 truncateSpecifiedLengthString="no" 
-       						 useNilForDefault="no">
+                   useNilForDefault="no"
```

Comment:
indentation issue. We should keep with the existing way right column to avoid changing every line.